src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => {
+  const personsPartner = `${person.sex === 'm' ? 'wife' : 'husband'}`;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+      <p className="Person__age">
+        I am
+        {' '}
+        {person.age}
```

Comment:
Use template string here as well like above
```suggestion
        {`I am ${person.age}`}
```